lib/rules/key-spacing.js

```diff
@@ -442,7 +442,8 @@ module.exports = {
                     isKeySide = side === "key",
                     isExtra = diff > 0,
                     diffAbs = Math.abs(diff),
-                    spaces = Array(diffAbs + 1).join(" ");
+                    // eslint-disable-next-line unicorn/no-new-array -- Inconsistent Unicorn errors
+                    spaces = new Array(diffAbs + 1).join(" ");
```

Comment:
`String#repeat()`?

Reply:
As above. Note you may want to see about reconciling the requiring of `new` from one rule by Unicorn and calling for it to be removed by another.